rpcs3/Emu/RSX/rsx_methods.cpp

```diff
@@ -41,7 +41,9 @@ namespace rsx
 	{
 		//Don't throw, gather information and ignore broken/garbage commands
 		//TODO: Investigate why these commands are executed at all. (Heap corruption? Alignment padding?)
-		LOG_ERROR(RSX, "Invalid RSX method 0x%x (arg=0x%x)", _reg << 2, arg);
+		const u32 start_cmd = vm::read32(rsx->ctrl->get); // TODO: get actual start cmd from fifo_ctrl instead (unreliable)
+		LOG_ERROR(RSX, "Invalid RSX method 0x%x (arg=0x%x, start=0x%x, count=0x%x, cmd=0x%x)", _reg << 2, arg, 
```

Comment:
also reporting raw cmd value as it also contains info if its a non-incrementing command or not, and other reserved bits values.